package.json

```diff
@@ -64,11 +63,16 @@
     "eslint": "3.6.0",
     "font-awesome": "4.6.3",
     "handlebars": "4.0.5",
+    "imports-loader": "0.6.5",
     "istanbul": "0.4.5",
+    "jquery": "2.1.1",
+    "jquery-ui": "1.12.1",
```

Comment:
NPM doesn't have 1.11.1 unfortunately, the closest are 1.10.5 and 1.12.0. The only thing we use from jquery-ui is sortable, so I'll check that it still works with 1.12.1 and stick with that if it works.